misc/net.minetest.minetest.metainfo.xml

```diff
@@ -0,0 +1,154 @@
+                bauen
+            </li>
+            <li xml:lang="fr">La possibilité de se promener, de creuser et de construire dans un monde voxel quasi-infini
+            </li>
+            <li>Crafting of items from raw materials</li>
+            <li xml:lang="de">Fertigen von Items aus Rohmaterialien</li>
+            <li xml:lang="fr">Fabrication d'objets à partir de matières premières</li>
+            <li>A simple modding API that supports many additions and modifications to the game</li>
+            <li xml:lang="de">Eine einfache Modding-API, die viele Ergänzungen und Änderungen am Spiel unterstützt</li>
+            <li xml:lang="fr">Une API de modding simple qui permet de nombreux ajouts et modifications au jeu</li>
+            <li>Multiplayer support via servers hosted by users</li>
+            <li xml:lang="de">Mehrspieler auf selber gehosteten Servern</li>
+            <li xml:lang="fr">Prise en charge du multijoueur via des serveurs hébergés par les utilisateurs</li>
+            <li>Beautiful lightning-fast map generator</li>
+            <li xml:lang="de">Wunderschöner, blitzschneller Kartengenerator</li>
+            <li xml:lang="fr">Générateur de cartes très rapide</li>
+        </ul>
+    </description>
+
+    <launchable type="desktop-id">net.minetest.minetest.desktop</launchable>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://www.minetest.net/media/gallery/1.jpg</image>
+        </screenshot>
+        <screenshot>
+            <image>https://www.minetest.net/media/gallery/3.jpg</image>
+        </screenshot>
+        <screenshot>
+            <image>https://www.minetest.net/media/gallery/5.jpg</image>
+        </screenshot>
+    </screenshots>
+
+    <icon type="stock">minetest</icon>
+
+    <categories>
+        <category>Game</category>
+        <category>Simulation</category>
+    </categories>
+
+    <keywords>
+        <keyword>sandbox</keyword>
+        <keyword>world</keyword>
+        <keyword>mining</keyword>
+        <keyword>crafting</keyword>
+        <keyword>blocks</keyword>
+        <keyword>nodes</keyword>
+        <keyword>multiplayer</keyword>
+        <keyword>roleplaying</keyword>
+    </keywords>
+
+    <url type="homepage">https://www.minetest.net</url>
+    <url type="bugtracker">https://www.minetest.net/get-involved/#reporting-issues</url>
+    <url type="translate">https://dev.minetest.net/Translation</url>
+    <url type="donation">https://www.minetest.net/get-involved/#donate</url>
+    <url type="faq">https://wiki.minetest.net/FAQ</url>
+    <url type="help">https://wiki.minetest.net</url>
+    <url type="vcs-browser">https://github.com/minetest/minetest</url>
+    <url type="contribute">https://www.minetest.net/get-involved</url>
+
+    <provides>
+        <binary>minetest</binary>
+    </provides>
+
+    <translation type="gettext">minetest</translation>
+
+    <update_contact>sfan5@live.de</update_contact>
```

Comment:
Please use this opportunity to put someone else's email here.
Volunteers?

Reply:
A Minetest mailing list would be convenient for such cases.